db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml

```diff
@@ -8818,6 +8818,17 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
                <arg>message</arg>
            </msg>
 
+           <msg>
+               <name>SE024</name>
+	       <text>Parameter cannot be null: {0}</text>
+               <arg>parameterName</arg>
+           </msg>
```

Comment:
redundant with

<msg>
               <name>TS006</name>
               <text>Parameter '{0}' cannot be null.</text>
               <arg>parameterName</arg>
           </msg>